src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_following_revert_in_createFiller.json

```diff
@@ -0,0 +1,74 @@
+{
+    "returndatacopy_following_revert_in_create" : {
+        "env" : {
+            "currentCoinbase" : "0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "0x0a00000000",
+            "currentNumber" : "0x01",
+            "currentTimestamp" : "0x03e8",
+            "previousHash" : "0x5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "expect" : [
+            {   "indexes" : {
+                   "data" : -1,
+                   "gas" : -1,
+                   "value" : -1
+                },
+                "network" : ["EIP150", "EIP158", "Frontier", "Homestead"],
+                "result" : {
+                    "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                        "storage" : {
+                            "0x00" : "0x01"
+                        }
+                    }
+                }
+            },
+            {   "indexes" : {
+                   "data" : -1,
+                   "gas" : -1,
+                   "value" : -1
+                },
+                "network" : ["Byzantium", "Constantinople"],
+                "result" : {
+                    "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                        "storage" : {
+                            "0x00" : "0x0000111122223333444455556666777788889999aaaabbbbccccddddeeeeffff"
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" : {
+            "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+				"balance" : "0x00",
+                "code" : "{ (seq (CREATE 0 0 (lll (seq (MSTORE 0 0x0000111122223333444455556666777788889999aaaabbbbccccddddeeeeffff) (REVERT 0 32)) 0)) (RETURNDATACOPY 0 0 32) (SSTORE 0 (MLOAD 0)) )}",
```

Comment:
how about putting the same but inside transaction init code?

Reply:
That doesn't work.  I want to do returndatacopy.

Reply:
I mean you could also put CREATE opcode inside transaction INIT code. 
and that new contract that being created from transaction init code would run the same as in your test here. and then you could check returndatacopy of that contract while being in transaction init code.

Reply:
I added a cell in the spreadsheet https://docs.google.com/spreadsheets/d/1xat7UI8GtB4ZGVdlK5_XQSHJZaMThi4SrlcL8XMZb5Q/edit#gid=1605675414&range=C59